seatunnel-engine/seatunnel-engine-client/pom.xml

```diff
@@ -68,7 +68,7 @@
         </dependency>
         <dependency>
             <groupId>org.apache.seatunnel</groupId>
-            <artifactId>connector-file-local</artifactId>
+            <artifactId>connector-console</artifactId>
```

Comment:
St-client depends on connector feeling strange

Reply:
just for test in client test module